src/Okta.Sdk.UnitTests/PayloadHandlerShould.cs

```diff
@@ -115,6 +118,8 @@ public void SetPemCertContent()
             pkixCertPayloadHandler.SetHttpRequestMessageContent(testHttpRequest, httpRequestMessage);
             httpRequestMessage.Content.Should().NotBeNull();
             httpRequestMessage.Content.GetType().Should().Be(typeof(ByteArrayContent));
+            httpRequestMessage.Content.Headers.Contains("Content-Type");
+            httpRequestMessage.Content.Headers.ContentType.ToString().Should().Be("application/x-pem-file");
```

Comment:
+1 to this addition! This is was I was talking about yesterday. Thanks for adding this.

Reply:
I was confused about how to correctly do this assertion, I figured it out thankfully.